Cargo.toml

```diff
@@ -39,6 +38,11 @@ tokio-codec = "0.1"
 tokio-io = "0.1"
 url = "1.7"
 
+[target.'cfg(windows)'.dependencies]
```

Comment:
This should be `cfg(target_os = "windows")` 

See https://github.com/sfackler/rust-native-tls/blob/master/Cargo.toml#L20 for an example that works.